demo/project_demo13-DebeziumPostgres/project.sql

```diff
@@ -0,0 +1,29 @@
+-- SQL program from the Debezium MySQL tutorial used to demo Feldera
+-- Debezium integration.
+
+CREATE TABLE customers (
+  id int NOT NULL PRIMARY KEY,
+  first_name varchar(255) NOT NULL,
+  last_name varchar(255) NOT NULL,
+  email varchar(255) NOT NULL
+) with ('materialized' = 'true');
```

Comment:
why do we need to materialize these tables?

Reply:
Just for debugging